src/app/loans/loans-view/repayment-schedule-tab/repayment-schedule-tab.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="container" >
 
   <div fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px" class="transaction-buttons" >
-    <button mat-raised-button color="primary" class="transaction-button" (click)="exportSchedule()">
+    <button mat-raised-button color="primary" class="transaction-button" (click)="exportSchedule()" *ngIf="loanId !== undefined">
```

Comment:
why not *ngIf="loanId" ?

Reply:
was using it to explicitly check for undefined value, I'll change to that for better readability. thanks Farooq